src/main/kotlin/org/stt/connector/jira/JiraConnector.kt

```diff
@@ -11,25 +10,20 @@ import javax.inject.Singleton
 @Singleton
 class JiraConnector @Inject
 constructor(configuration: JiraConfig) : Service {
-//    private val client: JiraClient?
-
-//    private val projectNames: Set<String> by lazy {
-//        internalGetProjectNames()
-//    }
+    private val client: JiraClient?
 
     init {
-//        val jiraURI = configuration.jiraURI
```

Comment:
I'm also ok with deleting this code

Reply:
I'm not sure what you by "also ok with deleting", this code is replaced with a new implemention to connect to Jira